mergify_engine/actions/assign.py

```diff
@@ -28,14 +29,35 @@ class AssignAction(actions.Action):
     validator = {
         voluptuous.Required("users", default=[]): [
             voluptuous.Coerce(types.Jinja2, types.GitHubLogin)
-        ]
+        ],
+        voluptuous.Required("add_users", default=[]): [
+            voluptuous.Coerce(types.Jinja2, types.GitHubLogin)
+        ],
     }
 
     silent_report = True
 
     def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Result:
+        users = list()
+        if self.config["users"]:
+            users = self._check_users(ctxt, self.config["users"])
+            self._add_assignees(ctxt, users)
+
+        if self.config["add_users"]:
+            users = self._check_users(ctxt, self.config["add_users"])
+            self._add_assignees(ctxt, users)
```

Comment:
it can return a `check_api.Result` failure that we need to return to the caller too.